discover/deployment/articles/02-configuring-liferay/02-document-repository.markdown

```diff
@@ -222,10 +222,11 @@ You must include this property in `system-ext.properties`:
     org.xml.sax.driver=com.sun.org.apache.xerces.internal.parsers.SAXParser
 
 Other app servers also need this configuration if they don't contain a
-`SAXParser`. 
+`SAXParser`. Remember to place the `system-ext.properties` in your @product@
+installation's `/WEB-INF/classes` directory.
 
```

Comment:
This is wrong, because it only applies to Tomcat. AFAIK, it should go in Liferay Home.